### PR TITLE
OVN ansible uninstall changes

### PR DESCRIPTION
**Pull request by pecameron**
Make the uninstall more complete

JIRA SDN-237 - make ansible uninstall more complete
https://jira.coreos.com/browse/SDN-237

Signed-off-by: Phil Cameron <pcameron@redhat.com>

### CODE REVIEW

**Comment by pecameron** [comment]
@dcbw @squeed @shettyg PTAL

File: dist/ansible/ovn-uninstall.yaml [hunk 1]
```diff
@@ -34,13 +34,23 @@
...
 # - name: Add adm policy auyuid
 #   shell: oc adm policy add-scc-to-user anyuid -z ovn
 #   ignore_errors: yes
 
 
+- hosts: nodes
+  become: yes
+  become_user: root
+  tasks:
+  - name: Remove remove the cni plugin config
```
Comment (dcbw) [inline dist/ansible/ovn-uninstall.yaml]: We probably just want to remove the config file we drop in there, not the whole directory...

Comment (pecameron) [reply dist/ansible/ovn-uninstall.yaml]: @dcbw We explicitly create the directory, and the file. That is why I deleted it. Would you like me to change it?

Comment (pecameron) [reply dist/ansible/ovn-uninstall.yaml]: @dcbw I made the change. PTAL

**Pull request merged**